server/adam/Adam.lean

```diff
@@ -25,37 +25,36 @@ Introduction
 "
 # Game Over oder QED?
 
-Willkommen zu unserem Prototyp eines Lean4-Lernspiels. Hier lernst du Computer-gestütztes
-Beweisen. Das Interface ist anfangs etwas vereinfacht, der \"Editor Mode\" funktioniert aber
-ziemlich gleich wie wenn du später Lean im VSCode benützt.
+Willkommen zu unserem Prototyp eines Lean4-Lernspiels. Hier lernst du computer-gestützte
+Beweisführung. Das Interface ist etwas vereinfacht, aber wenn du den *Editor Mode* aktivierst, fühlt es sich 
+fast genauso an wie eine professionalle IDE, etwa VSCode.
```

Comment:
```suggestion
fast genauso an wie in VSCode, der standard IDE für Lean.
```

Reply:
Da defacto nur "VSCode" und "emacs" ein Lean-Plugin haben, ist VSCode mehr oder weniger die einzige IDE, in der man Lean anständig nutzen kann.